echo-telemetry/src/main/java/com/netflix/spinnaker/echo/telemetry/TelemetryEventListener.java

```diff
@@ -217,4 +216,61 @@ public String toString() {
       return new String(getBytes(), StandardCharsets.UTF_8);
     }
   }
+
+  // Arbitrary container for TelemetryEventListener structured data, so there aren't huge
+  // fully-qualified names for the equivalent classes in the kork.proto package all over the code.
+  public static class TEL {
```

Comment:
Given that this is only used 3 times above, I'd suggest giving it a more descriptive name; reading `TEL` above seems more distracting than if we just gave it a longer name.  (Also, I think that this inner class can be `private` which would be particularly important if we're giving it a non-conventional name.)

Reply:
Renamed to `Holder`. Discussed in person that this needs to be `public` for deserialization.